tests/by-util/test_basenc.rs

```diff
@@ -26,7 +31,9 @@ fn test_invalid_input() {
     let error_message = if cfg!(windows) {
         "basenc: .: Permission denied\n"
     } else {
-        "basenc: error: invalid input\n"
+        // TODO
+        // Other implementations do not show " (os error 21)"
+        "basenc: read error: Is a directory (os error 21)\n"
```

Comment:
"os error 21" probably refers to the UNIX errno, and shouldn't appear in a platform-independent test. I suggest changing the error message and removing the "os error" entirely. After all, GNU base64 doesn't print it either, and we try to be as close to it as reasonably possible:

```console
$ base64 /tmp 
base64: read error: Is a directory
[$? = 1]
```